### PR TITLE
updating portifolio project 

### PR DESCRIPTION
**Pull request by Moh9998**
first update for this branch be ready for review then update it again

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Moh9998 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has some of the linters failing.Kindly fix the linter errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -39,12 +39,405 @@ <h1 class="name">
...
+                </div>
+            </section>
+
+            <section class="page1">
+
+                     <div class="geometry1">
+                    <img src="./images/image_geometry_2.png" alt="geometry">
+                  </div>
+                     <h1 class="projects">
+                    Projects
```
Comment (Kandy-Peter) [inline index.html:51]: - [ ] Please make sure you fix the button that comes above the `Project` title. This is due to the position it has that makes it overflow its container. Check how it looks:

image

File: index.html [hunk 2]
```diff
@@ -39,12 +39,405 @@ <h1 class="name">
...
+
+
+
+
+
+            <section class="page6">
+                <div class="about">
+                    <h1>
+                        About
+                    </h1>
```
Comment (Kandy-Peter) [inline index.html:297]: - [ ] Please, make sure The section `About` uses the font and colours (background and font) defined for the template.
**- Your design:**

image


**- Figma design:**

image

**Review by Kandy-Peter — CHANGES_REQUESTED** [review]
Hi @Moh9998,

Good job so far! I know you have worked hard on implementing the project and its requirements. However, there is an issue that you still need to work on to go to the next project but you are almost there!

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
## Required Changes ♻️
- [ ] Kindly check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Moh9998) [reply index.html:297]: when I use diffrent font family it cause linter errors 

how do i use other than sans,sarif??

**Comment by Moh9998** [comment]
I hope I fixed All The issues

**Review by sumairq — APPROVED** [review]
### Hi @Moh9998  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Moh9998 — COMMENTED** [review]
fainalizing the mobile version

**Pull request merged**